product_docs/docs/pgd/5/sequences.mdx

```diff
@@ -181,15 +181,15 @@ The differences between timeshard and SnowflakeId are as following:
 
 The globally allocated range (or `galloc`) sequences allocate ranges (chunks)
 of values to each node. When the local range is used up, a new range is
-allocated globally by consensus amongst the other nodes. This uses the key
+allocated globally by consensus among the other nodes. This behavior uses the key
 space efficiently but requires that the local node be connected to a majority
 of the nodes in the cluster for the sequence generator to progress when the
 currently assigned local range is used up.
 
 Unlike SnowflakeId sequences, `galloc` sequences support all sequence data types
 provided by PostgreSQL: `smallint`, `integer`, and `bigint`. This means that
 you can use `galloc` sequences in environments where 64-bit sequences are
-problematic. Examples include using integers in javascript, since that supports only
+problematic. Examples include using integers in JavaScript, since that supports only
 53-bit values, or when the sequence is displayed on output with limited space.
 
 The range assigned by each voting is currently predetermined based on the
```

Comment:
please verify the above sentence is correct. It looks like maybe a noun is missing either after "each" or after "voting"?

Reply:
each voting node

I believe.